enginerequests.go

```diff
@@ -183,6 +185,10 @@ func (e *Engine) ExecuteQuery(ctx context.Context, query *sdp.Query, items chan<
 
 	expanded := e.sh.ExpandQuery(query)
 
+	span.SetAttributes(
+		attribute.Int("ovm.discovery.numExpandedQueries", len(expanded)),
```

Comment:
All other attributes are `ovm.source` too:

```suggestion
		attribute.Int("ovm.source.numExpandedQueries", len(expanded)),
```